### ISSUE TITLE
Segmentation fault on linux

### ISSUE THREAD

**Issue by wykke** [ppy/osu#7025]
**Describe the bug:** 
when running the app, it closes giving segmentation fault, not even opening the game

**Screenshots or videos showing encountered issue:** 
![image](https://user-images.githubusercontent.com/38235118/69906845-9268da00-13a8-11ea-9200-8ab113612d32.png)

osu-lazer bash:
```
#!/bin/sh
export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib/osu-lazer"
dotnet '/usr/lib/osu-lazer/osu!.dll' "$@"
```

**osu!lazer version:** 
- 2019.1128.0
    - the `2019.1122.0` version is running fine

**Logs:** 
- [os.txt](https://github.com/ppy/osu/files/3907656/os.txt)
    - information about my os
- [installation.txt](https://github.com/ppy/osu/files/3907657/installation.txt)
    - installed using an [AUR](https://aur.archlinux.org/packages/osu-lazer/) script, edited by me

**Comment by wykke** [comment]
someone posted a better issue few minutes before me

**Comment by bdach** [comment]
See #7018, #7024 (most likely already resolved by ppy/osu-framework#3057).

By the way, since #2928 the `LD_LIBRARY_PATH` export should be no longer required.

**Comment by wykke** [comment]
thanks!
